query-node/mappings/content/utils.ts

```diff
@@ -518,10 +521,54 @@ export async function unsetAssetRelations(store: DatabaseManager, dataObject: St
     })
     await store.save<Video>(video)
 
+    // check if video is fully active
+    const isFullyActive = isVideoFullyActive(video)
+
+    // update video counters for channel, channel category, and video category if needed
+    if (wasFullyActive !== isFullyActive) {
+      await updateVideoActiveCounters(store, video, isFullyActive)
+    }
+
     // emit log event
     logger.info('Content has been disconnected from Video', {
       videoId: video.id.toString(),
       dataObjectId: dataObject.id,
     })
   }
+
+  // remove data object
+  await store.remove<StorageDataObject>(dataObject)
```

Comment:
I think this should either remain a separate action or this function should replace `removeDataObject` completely

Reply:
`removeDataObject` has been removed - fully replaced by `unsetAssetRelations`.